base.py

```diff
@@ -158,6 +160,13 @@ def _quote_free_identifiers(self, *ids):
         """
         return tuple([self.quote(i) for i in ids if i is not None])
 
+    def quote_schema(self, schema, force=None):
+        """
+        Split schema by a dot and merge with required quotes
+        """
+        idents = schema.split('.')
+        return '.'.join(self._quote_free_identifiers(*idents))
```

Comment:
By default, this method takes schema as one identifier. The fix here is split and merge with required quotes.